index.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" media="screen" href="./styles.css">
+    </head>
+
+    <body>
+        <section id="portfolio" class="project-section">
+            <div>
+                <div class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project1.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h1 class="title">Tonic</h1>
```

Comment:
- [ ] Kindly change this h1 to anh2, h3, ...it's recommended to have only one h1 by page for SEO reasons.

Reply:
ok thank